ods_ci/configs/templates/ldap/ldap.yaml

```diff
@@ -49,7 +49,8 @@ spec:
     spec:
       containers:
         - name: openldap
-          image: quay.io/croberts/openldapserver:latest
+          # Temporary solution until the image is automated and pushed to quay.io/modh
+          image: quay.io/rh-ee-jstetina/openldap-ocp@sha256:10233fef19f10b1b7d48f85e71faa064b98f422a984c522a86607ab35e56d21c
```

Comment:
Do we want to use it with the following vulnerabilities: 
Quay Security Scanner has detected 114 vulnerabilities.
 1 Medium-level vulnerabilities.
 83 Low-level vulnerabilities.
 30 Unknown-level vulnerabilities.

Reply:
Hi @kobihk , actually this is very good result - only 1 medium vulnerability.

Reply:
@apodhrad Can we expose this as user param and while running from jenkins, can we read this from vault or from jenkins repo?

Reply:
Its not clear to me what would be the param? I agree with @apodhrad , we should look to keep this simplified as QE and developers expect this to "just work".